src/main/resources/assets/lambdynlights/lang/it_it.json

```diff
@@ -0,0 +1,19 @@
+{
+  "lambdynlights.menu.light_sources": "Fonti di Luci Dinamiche",
+  "lambdynlights.menu.title": "Impostazioni di LambDynamicLights",
+  "lambdynlights.menu.canvas.1": "Disattivare luci dinamiche di entità è raccomandato con Canvas!",
+  "lambdynlights.menu.canvas.2": "L'illuminazione dinamica con Canvas è pesante sulla performance per il momento!",
```

Comment:
Anche qui l'ordine suona innaturale, oltre alla mancata traduzione da "performance" a "prestazioni". Consiglierei di riformulare la frase, visto che "prestazioni" e "pesante" non hanno lo stesso uso in italiano in questo caso.
Per questo consiglierei qualcosa come:
"Attualmente la luminosità dinamica è molto pesante con Canvas!"